docs/content/kv_commands.md

```diff
@@ -113,21 +174,26 @@ yes
 
 ### `list`
 
-Outputs a list of all keys in a given namespace. Optional params include
-1. `--prefix`: A prefix to filter listed keys
+Outputs a list of all keys in a given namespace. 
+
+Requires `--binding` or `--namespace-id` argument.
+
+Optional params include
+1. `--env`: The [environment](./environments) argument.
+1. `--prefix`: A prefix to filter listed keys.
 
 #### Usage
 The example below uses Python's JSON pretty-printing command line tool to pretty-print output.
 
 ```sh
-$ wrangler kv:key list f7b02e7fc70443149ac906dd81ec1791 --prefix="public" | python -m json.tool
+$ wrangler kv:key list --binding=MY_NAMESPACE --prefix="public" | python -m json.tool
```

Comment:
Do we want to recommend python's tool here? I think `jq` is more prevalent.

```console
wrangler kv:key list -b OKV | jq
[
  {
    "name": "one"
  }
]
```

Reply:
I'm just going to add pretty printing and a flag to do so: #541

Reply:
cool